sources/packages/backend/apps/api/src/services/application/application.service.ts

```diff
@@ -1874,36 +1878,60 @@ export class ApplicationService extends RecordDataModelService<Application> {
   /**
    * Get previous application versions for an application.
    * @param applicationId application id.
-   * @param options method options.
-   * - `limit` number of previous application versions to be returned.
-   * - `loadDynamicData` optionally loads the dynamic data.
    * @returns previous application versions.
    */
   async getPreviousApplicationVersions(
     applicationId: number,
```

Comment:
Please call `applicationId` as `parentApplicationId` here and also in the comment.